### DISCUSSION TITLE
Why heroku is suspending my account if I host this repository to deploy an app

### DISCUSSION THREAD

**Discussion by infographyc** [ratsan/Student-Database#4] [Q&A] [answered]
I've been trying to deploy an app from this repo for a while now. Everytime the deployment finishes heroku suspends my account after few hours. This is frustrating me.

**Comment by codehasan** [comment, marked as answer]
The possible reason is, this repo uses some library or code which is flagged as violation of terms of services in heroku. You should try checking flagged codes and remove them or check other projects to deploy an app.
